src/character.cpp

```diff
@@ -2113,21 +2113,26 @@ hp_part Character::body_window( const std::string &menu_header,
     };
 
     const auto hp_str = [precise]( const int hp, const int maximal_hp ) -> std::string {
-        if( hp <= 0 ) {
-            return "-----";
-        } else if( precise ) {
+        if( hp <= 0 )
+        {
+            return "==%==";
+        } else if( precise )
+        {
             return string_format( "%d", hp );
-        } else {
-            return string_format( "%-5s", get_hp_bar( hp, maximal_hp, false ).first );
+        } else
+        {
+            std::string h_bar = get_hp_bar( hp, maximal_hp, false ).first;
+            nc_color h_bar_col = get_hp_bar( hp, maximal_hp, false ).second;
+
+            return tag_colored_string( h_bar, h_bar_col ) + tag_colored_string( std::string( 5 - h_bar.size(), '.' ), c_white );
         }
     };
 
     uilist bmenu;
-    bmenu.text = menu_header;
-
-    // Add a header line so player can see the color of all other lines.
+    bmenu.desc_enabled = true;
+    bmenu.text = string_format( _( "Select a body part for: %s" ), menu_header );
```

Comment:
Just noticed that this line will also change the NPC wound menu message to `Select a body part for: <npcname>`. This can be a little misleading as IIRC that menu does not do anything when you select a body part. Maybe you can change the method signature to take a normal message and another message when there is no healable part instead?

Reply:
I've seen it while I was testing it, but somehow it seemed normal to me, but you are right, such messages can imply that you can select limbs for some action and that's misleading. I'll fix it.